docs/lib/geo/fragments/js/search.md

```diff
@@ -25,10 +25,8 @@ The location search UI component (`maplibre-gl-geocoder`) requires a "geocoding
 Finally, add the location search UI component (`MaplibreGeocoder`) to the map.
 
 ```javascript
-import { AmplifyMapLibreRequest, AmplifyGeocoderAPI } from "maplibre-gl-js-amplify";
+import { AmplifyMapLibreRequest, AmplifyGeocoderAPI, createAmplifyGeocoder } from "maplibre-gl-js-amplify";
```

Comment:
With this change, we should remove the import `AmplifyGeocoderAPI` as well as line 23 in this file _The location search UI component (`maplibre-gl-geocoder`) requires a "geocoding API"..._ and update line 25 to use `createAmplifyGeocoder`